apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/blogging-prompts-modal/index.js

```diff
@@ -0,0 +1,102 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
```

Comment:
yea probably not very much! I was just thinking as a default, but I don't think it's particularly useful here either

Reply:
Ugg, i tried this briefly and what a PITA. The TS linter gets (inconsistently) mad at things like the window object, using hook setters with values from the api, using dispatch in any way... and the fixes others seemed to have used in these areas feel like hacky work arounds (like making placeholder types simply to satisfy the linter). Since we didn't find how converting this to TS would be beneficial given its encapsulation, these silly problems it surfaces, etc. It really doesn't seem valuable to spend time on this.